mods/sgssb/moves.js

```diff
@@ -445,5 +445,20 @@ exports.BattleMovedex = {
         target: "Normal",
         type: "Electric",
     },
-
+    //CelestialTater
+     shellbreak: {
+     category: "Status",
+     id: "shellbreak",
+     isNonstandard: true,
+     name: "Shell Break",
+     boosts: {spa: 2, atk: 2, spe: 2, def: -1, spd: -1, accuracy: 1},
+     pp: 8,
```

Comment:
PP should be the base PP (5). PS will "use a PP Max" on the move moving it to 8 PP.